src/ripple/app/tx/impl/NFTokenMint.cpp

```diff
@@ -197,7 +197,10 @@ NFTokenMint::preclaim(PreclaimContext const& ctx)
             auto const nftIssuer =
                 ctx.tx[~sfIssuer].value_or(ctx.tx[sfAccount]);
 
-            if (!ctx.view.exists(keylet::line(nftIssuer, amount->issue())))
+            // If the IOU issuer and the NFToken issuer are the same,
```

Comment:
The commit message should be prefixed with `[FOLD]` to indicate that this commit should be merged (likely with the previous commit).